lib/bigquery-client/jobs.rb

```diff
@@ -6,7 +6,7 @@ def sql(query, options = {})
       result = jobs_query(query, options)
       names = result['schema']['fields'].map {|field| field['name'] }
       types = result['schema']['fields'].map {|field| field['type'] }
-      records = result['rows'].map {|row| row['f'].map {|record| record['v'] } }
+      records = if result['rows'] then result['rows'].map {|row| row['f'].map {|record| record['v'] } } else [] end
```

Comment:
I think the example below is simpler and better.

```ruby
records = (result['rows'] || []).map {|row| row['f'].map {|record| record['v'] }
```

What do you think about that?